recipes/scikit-cycling/meta.yaml

```diff
@@ -49,8 +46,8 @@ test:
     - skcycling
   commands:
     - pytest -l --pyargs skcycling -v -s
-    - conda inspect linkages -p $PREFIX scikit-learn  # [not win]
-    - conda inspect objects -p $PREFIX scikit-learn  # [osx]
+    - conda inspect linkages -p $PREFIX scikit-cycling  # [not win]
+    - conda inspect objects -p $PREFIX scikit-cycling  # [osx]
```

Comment:
Nice catch! 👍

Reply:
The output in the CI was not making sense :). I think this is good to go now.